rescompy/rescompy.py

```diff
@@ -411,7 +521,12 @@ def nrmse(self):
         error will be raised.
         Is only defined when target_outputs are provided."""
         
-        if self.target_outputs is not None:
+        if self.predict_length == 0:
```

Comment:
should these cause errors or just return nan?

Reply:
I thought the error message might be useful so that the user knows that they are trying to access something that is not defined. Maybe returning nan but also printing/logging a warning message would be better.